cmake/dependencies/tiff.cmake

```diff
@@ -0,0 +1,203 @@
+    POST_BUILD    
+    COMMENT "Installing ${_target}'s libs into ${RV_STAGE_LIB_DIR}"
+    COMMAND ${CMAKE_COMMAND} -E copy ${_base_dir}/src/libtiff/libtiff.lib ${_lib_dir}
+    COMMAND ${CMAKE_COMMAND} -E copy ${_base_dir}/src/libtiff/libtiff.dll ${_bin_dir}
+    COMMAND ${CMAKE_COMMAND} -E copy ${_base_dir}/src/libtiff/tiff.h ${_include_dir}
+    COMMAND ${CMAKE_COMMAND} -E copy ${_base_dir}/src/libtiff/tiffio.h ${_include_dir}
+    COMMAND ${CMAKE_COMMAND} -E copy ${_base_dir}/src/libtiff/tiffiop.h ${_include_dir}
+    COMMAND ${CMAKE_COMMAND} -E copy ${_base_dir}/src/libtiff/tiffio.hxx ${_include_dir}
+    COMMAND ${CMAKE_COMMAND} -E copy ${_base_dir}/src/libtiff/tiffvers.h ${_include_dir}
+    COMMAND ${CMAKE_COMMAND} -E copy ${_base_dir}/src/libtiff/tiffconf.h ${_include_dir}
+    COMMAND ${CMAKE_COMMAND} -E copy ${_base_dir}/src/libtiff/tif_config.h ${_include_dir}
+    COMMAND ${CMAKE_COMMAND} -E copy ${_base_dir}/src/libtiff/tif_dir.h ${_include_dir}
+
+    COMMAND ${CMAKE_COMMAND} -E copy ${_base_dir}/src/tools/fax2ps.exe ${_bin_dir}
+    COMMAND ${CMAKE_COMMAND} -E copy ${_base_dir}/src/tools/fax2tiff.exe ${_bin_dir}
+    COMMAND ${CMAKE_COMMAND} -E copy ${_base_dir}/src/tools/fax2ps.exe ${_bin_dir}
+    COMMAND ${CMAKE_COMMAND} -E copy ${_base_dir}/src/tools/fax2tiff.exe ${_bin_dir}
+    COMMAND ${CMAKE_COMMAND} -E copy ${_base_dir}/src/tools/pal2rgb.exe ${_bin_dir}
+    COMMAND ${CMAKE_COMMAND} -E copy ${_base_dir}/src/tools/ppm2tiff.exe ${_bin_dir}
+    COMMAND ${CMAKE_COMMAND} -E copy ${_base_dir}/src/tools/raw2tiff.exe ${_bin_dir}
+    COMMAND ${CMAKE_COMMAND} -E copy ${_base_dir}/src/tools/rgb2ycbcr.exe ${_bin_dir}
+    COMMAND ${CMAKE_COMMAND} -E copy ${_base_dir}/src/tools/thumbnail.exe ${_bin_dir}
+    COMMAND ${CMAKE_COMMAND} -E copy ${_base_dir}/src/tools/tiff2bw.exe ${_bin_dir}
+    COMMAND ${CMAKE_COMMAND} -E copy ${_base_dir}/src/tools/tiff2pdf.exe ${_bin_dir}
+    COMMAND ${CMAKE_COMMAND} -E copy ${_base_dir}/src/tools/tiff2ps.exe ${_bin_dir}
+    COMMAND ${CMAKE_COMMAND} -E copy ${_base_dir}/src/tools/tiff2rgba.exe ${_bin_dir}
+    COMMAND ${CMAKE_COMMAND} -E copy ${_base_dir}/src/tools/tiffcmp.exe ${_bin_dir}
+    COMMAND ${CMAKE_COMMAND} -E copy ${_base_dir}/src/tools/tiffcp.exe ${_bin_dir}
+    COMMAND ${CMAKE_COMMAND} -E copy ${_base_dir}/src/tools/tiffcrop.exe ${_bin_dir}
+    COMMAND ${CMAKE_COMMAND} -E copy ${_base_dir}/src/tools/tiffdither.exe ${_bin_dir}
+    COMMAND ${CMAKE_COMMAND} -E copy ${_base_dir}/src/tools/tiffdump.exe ${_bin_dir}
+    COMMAND ${CMAKE_COMMAND} -E copy ${_base_dir}/src/tools/tiffinfo.exe ${_bin_dir}
+    COMMAND ${CMAKE_COMMAND} -E copy ${_base_dir}/src/tools/tiffmedian.exe ${_bin_dir}
+    COMMAND ${CMAKE_COMMAND} -E copy ${_base_dir}/src/tools/tiffset.exe ${_bin_dir}
+    COMMAND ${CMAKE_COMMAND} -E copy ${_base_dir}/src/tools/tiffsplit.exe ${_bin_dir}
+  )
+
+ELSE()
+
+  # Linux & macOS
+  # Explicitely disabling LZMA support until we have updated build system and documentation
```

Comment:
This is from my initial import of the branch from Nicolas. I wonder what to do with disabling this.